File: order_backend_springboot/src/main/java/com/order/domain/entity/OrderDetail.java
package com.order.domain.entity;

import lombok.Data;
import org.hibernate.annotations.DynamicUpdate;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.util.Date;

@Entity
@Data
@DynamicUpdate
public class OrderDetail {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String orderNo;

    private Integer goodsId;

    private Integer count = 1;

    private Integer status = 0;

    private Date createTime;

    private Date updateTime;
}

File: order_backend_node/app/model/orderDetail.js
'use strict';

module.exports = app => {
  const { INTEGER, STRING } = app.Sequelize;
  return app.model.define('order_detail', {
    id: { type: INTEGER, primaryKey: true, autoIncrement: true },
    orderNo: STRING,
    goodsId: INTEGER,
    count: INTEGER,
    status: INTEGER,
  }, {
    freezeTableName: true,
  });
};
